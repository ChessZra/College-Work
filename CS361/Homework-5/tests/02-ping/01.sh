#!/bin/bash

PORT=$@

EOL=$'\r\n'
REQUEST=$'GET /ping HTTP/1.1'${EOL}${EOL}

printf "$REQUEST" | nc 127.0.0.1 $PORT
