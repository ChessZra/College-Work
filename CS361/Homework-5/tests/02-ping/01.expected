HTTP/1.1 200 OK
Content-Length: 4

pong